test/test_splitmix64.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2010 Steven Watanabe
+ * Copyright (c) 2022 Matt Borland
+ * Distributed under the Boost Software License, Version 1.0. (See
+ * accompanying file LICENSE_1_0.txt or copy at
+ * http://www.boost.org/LICENCE_1_0.txt)
+ *
+ */
+
+#include <cstdint>
+#include <array>
+#include <limits>
+#include <boost/core/lightweight_test.hpp>
+#include <boost/random/splitmix64.hpp>
+
+int main(void)
+{
+    constexpr auto const_min = (std::numeric_limits<std::uint64_t>::min)();
+    constexpr auto const_max = (std::numeric_limits<std::uint64_t>::max)();
+    
+    boost::random::splitmix64 rng;
+    
+    double entropy = rng.entropy();
```

Comment:
I confess to not understanding the `.entropy()` member here; is this the entropy of a uniform distribution?

Reply:
Since this is a [PRNG](https://en.cppreference.com/w/cpp/numeric/random/random_device/entropy) it is always 0 so not overly useful. Removed.